packages/react/src/components/DropdownMenu/DropdownMenuContent.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { forwardRef, useContext, useRef } from 'react';
+import {
+  useFloating,
+  autoUpdate,
+  offset,
+  useClick,
+  useDismiss,
+  useFocus,
+  useInteractions,
+  useMergeRefs,
+  useRole,
+  shift,
+  FloatingFocusManager,
+  FloatingPortal,
+} from '@floating-ui/react';
+import cl from 'clsx';
+
+import { useIsomorphicLayoutEffect } from '../../hooks';
+
+import classes from './DropdownMenu.module.css';
+import { DropdownMenuContext } from './DropdownMenu';
+
+const GAP = 4;
+
+export type DropdownMenuContentProps = {
+  children: React.ReactNode;
+} & React.HTMLAttributes<HTMLUListElement>;
+
+export const DropdownMenuContent = forwardRef<
+  HTMLUListElement,
+  DropdownMenuContentProps
+>(({ className, children, ...rest }, ref) => {
+  const {
+    size,
+    placement,
+    portal,
+    anchor,
+    isControlled,
+    internalOpen,
+    setInternalOpen,
+    onClose,
+  } = useContext(DropdownMenuContext);
+
+  const Container = portal ? FloatingPortal : React.Fragment;
+  const floatingEl = useRef<HTMLUListElement>(null);
```

Comment:
Might aswell fix this also -> `floatingRef`

Reply:
`Popover` uses `floatingEl`, so I think we should use this here as well

Reply:
Aight, lets fix it in a chore afterwards 👍